src/actions/coursesActions.js

```diff
@@ -1,22 +1,18 @@
+/* global is_demo */
+
 import { get_courses } from '../api/api'
-const courses_url = 'http://localhost:8082/api/courses'
 
-export function fetch_courses(new_term) {
-  return function(dispatch) {
+const courses_url = '/v1/courses/'
+
+export function fetch_courses(term) {
+  return function (dispatch) {
     dispatch({ type: 'FETCH_COURSES_START', payload: {} })
-    get_courses(new_term, courses_url)
-      .then(courses => {
-        dispatch({ type: 'RECEIVE_COURSES', payload: courses })
+    get_courses(is_demo, term, courses_url)
```

Comment:
I actually don't know how other people feel about this. But. 

In some other projects, `IS_DEMO` is defined as a constant in the `public/index.html` file and then in the soffit's `view.jsp`. Then in the `app.js` file it's just a global variable. The function arguments don't include `is_demo`, but the check still happens based on the global variable being pulled in. I don't know if people have a preference either way, but it might be nice to get some feedback so all our repos can do it the same way

Reply:
maybe we can bring this up in our standards meeting on the 31st?

Reply:
That sounds good to me